### PR TITLE
Building and Room Number

### PR DESCRIPTION
**Pull request by Hls5235**
This will allow the building and room number to be displayed neatly on a mobile device and a computer when viewing the schedule. This should help students see building and room number of their classes because most do not know you can click to view that. I am worried about using '\u00A0' on line 103, so if their is any alternative please let me know so I can change that.

### CODE REVIEW

File: src/services/schedule.ts [hunk 1]
```diff
@@ -30,6 +30,19 @@ type DbSchedule = {
...
+  }
+}
+
+function mobileFormat() {
+  if (detectMobile() == false) {
+    return ' | ';
+  } else {
+    return ' ';
+  }
+}
```
Comment (amos-cha) [inline src/services/schedule.ts:45]: You only use these functions once and I'm not sure if 1514px equates to mobile. Seems oddly wide.
```suggestion
function mobileFormat() {
  return window.innerWidth >= 1514 ? ' | ' : '';
}
```

**Review by amos-cha — CHANGES_REQUESTED** [review]

File: src/components/Profile/components/SchedulePanel/index.tsx [hunk 2]
```diff
@@ -23,6 +23,10 @@ type Props = {
   myProf: boolean;
 };
 
+export function mobileFormat() {
+  return window.innerWidth >= 1514 ? ' | ' : ' ';
+}
+
```
[the remark this answers is not in the document]
Comment (Hls5235) [reply src/components/Profile/components/SchedulePanel/index.tsx]: Looks like I didn't save the file, sorry.

File: src/components/Profile/components/SchedulePanel/components/ScheduleCalendar/index.tsx [hunk 3]
```diff
@@ -9,7 +9,19 @@ type Props = {
...
+  }
+}
 
+export function mobileFormat() {
+  if (detectMobile() == false) {
+    return ' | ';
+  } else {
+    return ' ';
+  }
+}
```
[the remark this answers is not in the document]
Comment (Hls5235) [reply src/components/Profile/components/SchedulePanel/components/ScheduleCalendar/index.tsx]: Looks like I didn't save the file, sorry.

**Review by amos-cha — APPROVED** [review]
lgtm

**Pull request merged**